### ISSUE TITLE
Trouble when convert md with a PNG image to docx

### ISSUE THREAD

**Issue by DingJunyao** [jgm/pandoc#4612]
Version: 2.2
example: 

```markdown
![bug img](./bug.png)
```

![bug.png](https://user-images.githubusercontent.com/11372753/39426821-e6bd16b0-4cb3-11e8-9058-069a1c2cd9de.png)

(I don't know if the image is compressed, but there is a zip file containing the md file and image: 
[bug.zip](https://github.com/jgm/pandoc/files/1960485/bug.zip)
)

I create md file including it, use pandoc to convert to docx:

```shell
pandoc ./1.md -o ./1.docx
```

Nothing wrong on the terminal. But error happens when I open docx: 

![error message](https://user-images.githubusercontent.com/11372753/39426768-adf0d9ca-4cb3-11e8-8121-f8d152a6cab1.PNG)

(Sorry, unable to open 1.docx, for there is a problem in the content. 
Details:
There is no details for the error
Position: Part: /word/document.xml, Line 0, Col 0)

I tried "open and repair", then the picture disappeared, remaining caption of the picture. 

![2](https://user-images.githubusercontent.com/11372753/39427130-3ee4ffa0-4cb5-11e8-815d-09b5f9e00ffa.PNG)

When I convert the picture to JPEG format with Paint, and change the path in the md file, it's normal. However, when I convert it to PNG format again and change the path in the md file, same error happens. So it may be fault of Pandoc.

**Comment by jkr** [comment]
Hmm, a couple of observations:

1. other pngs seem to work (not sure if it's the size of the image, the alpha layer, or something else. Will do further investigation).

2. This png works with the pptx writer.

So this writer works with some pngs but not this one, and this png works with some ooxml writers but not this one. Will report more as I get into it.

**Comment by LukeTOBrien** [comment]
Hello,

Word 2007
Pandoc: 2.2.3.2

I have this issue too and it seems to be for any image, I have tried gif, png and jpg.  
I can't even open and repair.

I have opened the .docx as a zip and the image seems to be in the media folder okay and the xml.rels look okay too.  
When I upload to Google Drive I can see the image in the document okay.... but then when I download and open in Word I get the same issue (presume no change in file).

Is there a slight change between the 2007 Open XML format and the (latest) 2017 one?
